hr_holidays_accrual_by_tag/models/hr_holidays_accrual_by_tag.py

```diff
@@ -0,0 +1,65 @@
+from odoo import api, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    @api.model
+    def create(self, vals):
+        employee = super(HrEmployee, self).create(vals)
```

Comment:
chore(non-blocking):
```suggestion
        employee = super().create(vals)
```